### DISCUSSION TITLE
Show the size of selected directories

### DISCUSSION THREAD

**Discussion by mortezadadgar** [jarun/nnn#1952] [Q&A]
Hey,
Currently `S` returns 0B for size of directory, I can sort disk usage but that won't show size of selected directories.

**Comment by jarun** [comment]
It is working as designed. traversing directories can be slow. We wanted to avoid that with `S`.

**Reply by mortezadadgar** [reply]
So there's no way to know their size?

**Reply by jarun** [reply]
Please use `du` mode.
